File: frontend/src/app/demo/api/dashboard/dashboardInfo.ts
export class DashboardInfo {
    orders? :number;
    percentageOrders? :number;
    revenue? :number; 
    percentageRevenue? :number;
    sub? :number;
    percentageSub? :number;

    constructor(){
        this.orders=0;
        this.percentageOrders=0;
        this.revenue=0;
        this.percentageRevenue=0;
        this.sub=0;
        this.percentageSub=0;
    }
}

File: backend/src/main/java/com/lms/model/stripe/DashboardInfo.java
package com.lms.model.stripe;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DashboardInfo {

    private int orders;
    private double percentageOrders;
    private double revenue;
    private double percentageRevenue;
    private int Sub;
    private double percentageSub;
}
